DEVELOPMENT.md

```diff
@@ -78,15 +72,17 @@ _Adding the `upstream` remote sets you up nicely for regularly
 Once you reach this point you are ready to do a full build and deploy as
 follows.
 
-# Deploy core configurations and Kafka
+### Set up a Kubernetes cluster
 
-```bash
-# re-execute the script in case some errors appear. 
-# (this can happen when the CRDs hasn't been registered and we try to create a Kafka cluster)
-./test/kafka/kafka_setup.sh 
+- This guide assumes you have a Kubernetes cluster up and running. - https://kubernetes.io/docs/setup/
 
-kubectl apply -f config
-```
+1. Execute `source test/e2e-common.sh `
```

Comment:
there is a space before the end of the code block

Reply:
Fixed.

What do you think about re-using test script, for development set up? I'm using it daily and it works really well; 
it removes the need for every developer to create its own 5min script.